src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,21 +1,35 @@
 package edu.kis.vh.nursery;
 
-public class FIFORhymer extends defaultCountingOutRhymer {
+import edu.kis.vh.nursery.storage.IntArrayStack;
+import edu.kis.vh.nursery.storage.IntStorage;
+
+public class FIFORhymer extends DefaultCountingOutRhymer {
+
+	/*** 
+	 * ! Odpowiedz na Pytanie: Jaki wybór będzie najlepszy (napisz komentarz)? 
+	 * W celu optymalizacji lepszym wyborem będzie IntArrayStack, poprzez wykorzystanie statycznej tablicy i 
+	 * braku potrzeby alokacji pamięci na nowy Node skraca się czas dostępu potrzebny do przypisania nowej wartości.
```

Comment:
true, but unfortunately, it has a limited size and it may turn out to be a problem
We should be in interested the most reliable implementation.